### DISCUSSION TITLE
Comando arresta sparito (HA su docker)

### DISCUSSION THREAD

**Discussion by manny199** [indomus/forum#1244] [GENERALE - Discussioni generiche]
Ciao a tutti,
dopo avere aggiornato HA raspbian OS su docker è sparito il pulsante arresta.
![immagine](https://user-images.githubusercontent.com/50017898/147695607-4d25b770-cc87-4c02-abff-cd8c105077b1.png)
Come risolvo?

**Comment by snakuzzo** [comment]
L'hanno rimosso.
[QUI](https://github.com/home-assistant/core/issues/62111) trovi la spiegazione di Frenck

Credimi che per la tua installazione è completamente inutile...
